src/ui/components/shared/UserSettingsModal/ExperimentalSettings.tsx

```diff
@@ -50,11 +50,6 @@ const EXPERIMENTAL_SETTINGS: ExperimentalSetting[] = [
     key: "multipleControllerUseSnapshots",
     secret: true,
   },
-  {
-    label: "Code Heatmaps ",
-    description: "Calculate hit counts for editor files all at once",
-    key: "codeHeatMaps",
-  },
```

Comment:
lets keep this here for a bit. There might be some inconsistencies where we can recommend to users that they disable it on a one off basis